engine/src/cubos/engine/tools/world_inspector/plugin.cpp

```diff
@@ -0,0 +1,42 @@
+#include <cubos/engine/tools/world_inspector/plugin.hpp>
+#include <imgui.h>
+
+#include <cubos/engine/plugins/imgui.hpp>
+#include <cubos/engine/plugins/tools/entity_selector.hpp>
+
+using namespace cubos;
+using namespace cubos::engine::plugins::tools;
+using namespace cubos::core::ecs;
```

Comment:
Change this to something like:

```
using cubos::core::ecs::World;
using namespace cubos::engine; // <- okay because this belongs to the engine module anyway